dev/devicelab/bin/tasks/plugin_test_win.dart

```diff
@@ -9,7 +9,7 @@ import 'package:flutter_devicelab/framework/framework.dart';
 
 Future<void> main() async {
   await task(combine(<TaskFunction>[
-    PluginTest('apk', <String>['-a', 'java']),
-    PluginTest('apk', <String>['-a', 'kotlin']),
+    PluginTest('apk', <String>['-a', 'java', '--platforms=java']),
+    PluginTest('apk', <String>['-a', 'kotlin', '--platforms=java']),
```

Comment:
`--platforms=android`?

Reply:
done. The test probably didn't even check for specific plugin folder, thats why it passed?